### PR TITLE
Added new shortcuts

### PR DESCRIPTION
**Pull request by zapacni**
Active forum users may find that they are often flagging duplicates, for instance in Scripting Support, "how do i script" is asked daily. !!DUPE is to tell a user yourself in a DM that they should search; ~~!!DUPEFLAG~~ !!FLAGDUPE is for when you want to flag the user.

### CODE REVIEW

**Comment by zapacni** [comment]
Feel free to suggest alternative wording as well since I felt it could be improved somewhere.

**Comment by FxllenCode** [comment]
Looks pretty good. (And I'll open a new issue for this,) but you can't use these shortcuts by clicking message from their name. I might be able to fix but I'll see.

**Comment by zapacni** [comment]
You can, but you just need to clear everything first, since when you click on message from their name in a post, it adds a link to their post at the beginning

**Comment by FxllenCode** [comment]
Huh. Strange, but your right. Thanks!

**Review by posatta — CHANGES_REQUESTED** [review]
When you're typing !!DUPEFLAG, it'd autofill for !!DUPE. You might want to change it to !!FLAGDUPE or something.

File: src/templates.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 const shortcuts = {
+	"!!DUPE": "Many topics have been made regarding this; in the future please try to search before posting.\n", // the new line is so you can link the search query
+	"!!FLAGDUPE": "Duplicate of: ", // for when you flag; space is for adding search query
```
Comment (FxllenCode) [inline src/templates.js:3]: Probably should set it so it makes a new line using /n, would look cleaner.

**Review by FxllenCode — CHANGES_REQUESTED** [review]
For !!FLAGDUPE, make a new line with /n so it looks cleaner and links to make it easy for DET.

**Comment by ElliottLMz** [comment]
`!!FLAGDUPE` is harder to write than `Duplicate of:`. I want to keep the number of templates to a relative minimum so I don't feel a need for FLAGDUPE.

**Review by ElliottLMz — CHANGES_REQUESTED** [review]
!!FLAGDUPE is harder to write than Duplicate of:. I want to keep the number of templates to a relative minimum so I don't feel a need for FLAGDUPE.

**Comment by zapacni** [comment]
I went ahead and removed !!FLAGDUPE. Thanks

**Review by FxllenCode — APPROVED** [review]
Looks good to me- I'd say it's ready.

**Review by ElliottLMz — APPROVED** [review]

**Pull request merged**